Software/DumptruckUltra-MCU/hw/i2c/i2cBusManager.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * i2cBusManager.hpp
+ *
+ *  Created on: Mar 27, 2023
+ *      Author: surao
+ */
+
+#ifndef I2CBUSMANAGER_HPP_
+#define I2CBUSMANAGER_HPP_
+
+#include "cy_pdl.h"
+#include "cybsp.h"
+#include "cyhal.h"
+#include "i2cBusManager.hpp"
+#include <stdio.h>
+
+/* Public API */
+class I2CBusManager {
```

Comment:
For cleanliness and organization this should go to a namespace like `Hardware::I2C`.